js/ui/tree_view/ui.tree_view.base.js

```diff
@@ -697,6 +699,9 @@ const TreeViewBase = HierarchicalCollectionWidget.inherit({
 
         if(this._hasCustomExpanderIcons()) {
             $node.addClass(ITEM_WITH_CUSTOM_EXPANDER_ICON_CLASS);
+            if(this.option('rtlEnabled')) {
+                $nodeContainer.addClass(RTL_CUSTOM_EXPANDER_ICON_CONTAINER_OFFSET_CLASS);
```

Comment:
Cannot we always add this class, but change the selector in `.scss` to smth like
```
.dx-rtl {
    .dx-treeview-custom-expander-icon-item-container {
    // ...
    }
}
```

Reply:
yes we can